robotCompletionFinal.cpp

```diff
@@ -0,0 +1,143 @@
+#include "ImageProcessor.hpp"
+
+int core(){
+	double vLeft = 40.0;
+    double vRight = 30.0;
+    char v_go = 20;
+    double Kp = 0.1;
+    char dv = 0;
```

Comment:
Since these variables are more or less similar across Core & Completion, don't you reckon we can put them outside the functions? That way you wouldn't have to declare them twice. Also, if a value is different for each function, it can just be changed inside the function (e.g. Kp)